src/device/main_m4/src/progblobs.cpp

```diff
@@ -44,16 +44,30 @@ static uint32_t getTxData(uint8_t *data, uint32_t len)
     return blobs_.getBlock(data, len);
 }
 
+static void enable_logging(bool enable)
+{
+    static bool sd_card_header_intialized = false;
+
+    // Only update the SD Card's header block once on first enable.
+    if (!sd_card_header_intialized && enable)
+    {
+        sdmmc_updateHeader();
+        sd_card_header_intialized = true;
```

Comment:
should `sd_card_header_intialized` be set to the value returned by `sdmmc_updateHeader()` since it may return false?

Reply:
Good catch. Done